### ISSUE TITLE
Inference Support - Ubuntu 18.04 (HIP/OpenCL)

### ISSUE THREAD

**Issue by kiritigowda** [GPUOpen-ProfessionalCompute-Libraries/MIVisionX#849]
Ubuntu `18.04` default Python3 version `3.6.X` does not support pip3 install `protobuf onnx`. This does not allow `CAFFE` & `ONNX`  model conversions.

Continued support for U18 inference needs additional steps for both HIP/OpenCL. Should we move to Ubuntu `22.04` going forward?

```
python3 --version
Python 3.6.9
```

```
Collecting google
  Downloading https://files.pythonhosted.org/packages/ac/35/17c9141c4ae21e9a29a43acdfd848e3e468a810517f862cad07977bf8fe9/google-3.0.0-py2.py3-none-any.whl (45kB)
    100% |################################| 51kB 7.5MB/s 
Collecting protobuf
  Downloading https://files.pythonhosted.org/packages/6c/be/4e32d02bf08b8f76bf6e59f2a531690c1e4264530404501f3489ca975d9a/protobuf-4.21.0-py2.py3-none-any.whl (164kB)
    100% |################################| 174kB 9.4MB/s 
protobuf requires Python '>=3.7' but the running Python is 3.6.9
Collecting onnx
  Downloading https://files.pythonhosted.org/packages/fd/b7/fccddff4d1873074605ff08acc812202b4a849cf4925b1f6ed5eeba575c4/onnx-1.11.0.tar.gz (9.9MB)
    100% |################################| 9.9MB 199kB/s 
Requirement already satisfied: numpy>=1.16.6 in /usr/local/lib/python3.6/dist-packages (from onnx)
Collecting protobuf>=3.12.2 (from onnx)
  Using cached https://files.pythonhosted.org/packages/6c/be/4e32d02bf08b8f76bf6e59f2a531690c1e4264530404501f3489ca975d9a/protobuf-4.21.0-py2.py3-none-any.whl
protobuf requires Python '>=3.7' but the running Python is 3.6.9
```

**Comment by kiritigowda** [comment]
@rrawther @asalmanp -- For Ubuntu `18.04` need to use

```
pip3 install protobuf==3.12.2
```

**Comment by kiritigowda** [comment]
Issue resolved with PR #850
